examples/db-templates/mongodb-ephemeral-template.json

```diff
@@ -152,13 +155,21 @@
   ],
   "parameters": [
     {
+      "name": "MEMORY_LIMIT",
+      "displayName": "Memory limit",
+      "description": "Maximum amount of memory a pod can use",
```

Comment:
Yeah, I have seen a (new) user perspective a lot in interchanging between pod and container with the config here.

I'll switch.